oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBIndexInstaller.java

```diff
@@ -67,13 +72,13 @@ protected boolean isExists(Model model) throws StorageException {
     protected void createTable(Model model) throws StorageException {
         try {
             if (model.isTimeSeries() && model.isRecord()) { // stream
-                Stream stream = (Stream) MetadataRegistry.INSTANCE.registerModel(model, config);
+                Stream stream = (Stream) MetadataRegistry.INSTANCE.registerModel(model, config, configService);
                 if (stream != null) {
                     log.info("install stream schema {}", model.getName());
                     ((BanyanDBStorageClient) client).define(stream);
                 }
             } else if (model.isTimeSeries() && !model.isRecord()) { // measure
-                Measure measure = (Measure) MetadataRegistry.INSTANCE.registerModel(model, config);
+                Measure measure = (Measure) MetadataRegistry.INSTANCE.registerModel(model, config, configService);
```

Comment:
I think we should separate `registerStreamModel` and `registerMeasureModel` to keep API clear, especially for the `return type`.

And where would we create the object when `!model.isTimeSeries()`? Or it is not required to be pre-created?

Reply:
If it isn't required, please add comments here to make this clear.

Reply:
We will create a validation to check it as https://github.com/apache/skywalking/issues/9902. Please add comments here.